### DISCUSSION TITLE
Who is Vercel's Certificate authority?

### DISCUSSION THREAD

**Discussion by JithinKS97** [vercel/community#45] [Help] [answered]
We are trying to build an app that requires integration with a public portal. The problem is in order to integrate the app, they have the requirement that the certificate authority should be from a specified list. So I would like to know if the CA for Vercel is included in that list? Can someone please tell me this, or point to any source where I can get more details on this.

**Comment by khaiyuanyap** [comment, marked as answer]
According to the DNS records listed, the CA is Let's Encrypt. 
![image](https://user-images.githubusercontent.com/83793501/145581629-340fe83d-2075-47d3-b800-ba5767f8eaf2.png)

Additional sources:
https://vercel.com/blog/automatic-ssl-with-vercel-lets-encrypt
